master/internal/core.go

```diff
@@ -459,38 +459,13 @@ func (m *Master) getResourceAllocations(c echo.Context) error {
 		Where("ts.event_type = 'IMAGEPULL'").
 		Group("a.allocation_id")
 
-	taskWorkspaceIDs := db.Bun().NewSelect().
```

Comment:
This is just
![chef_kiss](https://github.com/determined-ai/determined/assets/31023784/80acf22d-5c8d-4279-94bc-80d4705f3fcf)

Love to see this removed